application/src/main/java/org/thingsboard/server/actors/calculatedField/CalculatedFieldManagerMessageProcessor.java

```diff
@@ -68,22 +76,30 @@ public class CalculatedFieldManagerMessageProcessor extends AbstractContextAware
 
     private final CalculatedFieldProcessingService cfExecService;
     private final CalculatedFieldStateService cfStateService;
-    private final CalculatedFieldEntityProfileCache cfEntityCache;
     private final CalculatedFieldService cfDaoService;
+    private final DeviceService deviceService;
+    private final AssetService assetService;
     private final TbAssetProfileCache assetProfileCache;
     private final TbDeviceProfileCache deviceProfileCache;
+    private final TenantEntityProfileCache entityProfileCache;
     protected final TenantId tenantId;
 
     protected TbActorCtx ctx;
 
+    @Value("${calculated_fields.init_fetch_pack_size:50000}")
```

Comment:
Please introduce new property init_tenant_fetch_pack_size with the default value of 1000